deploy/marlowe-runtime/templates/marlowe-web-server.yaml

```diff
@@ -34,6 +34,11 @@ spec:
           port: 3780
           protocol: TCP
       traits:
+      - type: cpuscaler
+        properties:
+          min: 1
+          max: 10
+          cpuUtil: 50
```

Comment:
We can't scale the webserver until we address the in-memory state issue

Reply:
Disabled on [3b3cb48](https://github.com/input-output-hk/marlowe-cardano/pull/740/commits/3b3cb483b96c2b531875195719ebaec8c2fa5e02)